slides/aml-06-linear-models-classification/index.html

```diff
@@ -1347,19 +1347,22 @@
 
 .smaller[
 ```
-gamma : float, optional (default = "auto")
+gamma{‘scale’, ‘auto’} or float, optional (default=’scale’)
```

Comment:
shouldn't there be a colon?

Reply:
Interesting how copy and paste directly from the HTMl does not capture the `:`.